assets/js/modules/analytics/util/report-date-range-args.test.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Tests for reporting date range utilities.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { generateDateRangeArgs } from './report-date-range-args';
+
+describe( 'Analytics reporting date range arguments', () => {
+	describe( 'generateDateRangeArgs', () => {
+		const allArgs = {
+			startDate: 'start-date',
+			endDate: 'end-date',
+			compareStartDate: 'compare-start-date',
+			compareEndDate: 'compare-end-date',
```

Comment:
Are these values reflective of the actual ones used? Wouldn't they be `YYYY-MM-DD` or something like that?

Let's use actual values to test this so it's easier for the next developer to read and understand the motivation of the test. Technically speaking it's the same if it's letters or numbers, but it better conveys the intent and improves readability of the test for it to look like real data.